test/e2e/airgap.go

```diff
@@ -22,3 +22,15 @@ func runVSphereAirgapConfigFlow(test *framework.ClusterE2ETest, localCIDRs strin
 	)
 	test.DeleteCluster()
 }
+
+func runDockerAirgapConfigFlow(test *framework.ClusterE2ETest) {
+	test.GenerateClusterConfig()
+	test.DownloadArtifacts()
+	test.ExtractDownloadedArtifacts()
+	test.DownloadImages()
+	test.ImportImages()
+	test.ChangeInstanceSecurityGroup("sg-0f3b5c1368db406ee")
```

Comment:
Yet another environment variable. But I am not against it, I think it's good to have multiple ways to airgap the network

Reply:
Ahh this is just for E2E tests, this simulates the situation where the admin machine might be initially open to the internet for importing artifacts but airgapped after that.